site/docs/guidelines/workflow.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Workflow
+navTitle: Workflow
+summaryParagraph: A workflow is a long task broken up into a series of smaller, linked steps.
+tags: ["Wizard", "Multi-step workflow", "Onboarding"]
+needToKnow:
+  - "Use good defaults everywhere possible."
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D49154%253A36853&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Breaking the long task into a series of smaller, linked steps, lets you:
+
+- Distribute the overall cognitive load
+- Help the user understand the process by breaking it into more understandable chunks
+- Guide the user through the process with minimal friction.
+- Allow decisions made in preceding steps to vary the workflow of subsequent steps 
```

Comment:
If there is conditional branching, is the progress bar dynamic? Should it change based on user inputs, or always displays the most/least amount of steps?